Survey/Configurations/Environment/BuildConfiguration.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  BuildConfiguration.swift
+//  Survey
+//
+//  Created by Khanh on 01/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+
+final class BuildConfiguration {
```

Comment:
If you use `SWIFT_ACTIVE_COMPILATION_CONDITIONS` you can define this logic in the following way. That way you can have a table of configurations, not just enum. Just an optional suggestion. https://nimblehq.co/compass/development/ios/project-configuration/#active-compilation-conditions
```
import Foundation

enum Environment {

    static func based<T>(staging: T, uat: T, production: T) -> T {
        #if PRODUCTION
        return production
        #elseif UAT
        return uat
        #else
        return staging
        #endif
    }
}
```

Reply:
That's a great approach with less code and more convenient 💪 , I fixed at https://github.com/nkhanh44/ic-survey-ios/commit/2809248129b9cadb424df423d57d6fdf3722b046.